UnitystationLauncher/Models/Installation.cs

```diff
@@ -65,54 +58,55 @@ public Installation(string folderPath)
             return exe;
         }
 
-        public void StartImp()
+        public void Start(IPAddress ip, short port, string? refreshToken, string? uid)
+        {
+            Start($"--server {ip} --port {port} --refreshtoken {refreshToken} --uid {uid}");
+        }
+
+        public void Start()
```

Comment:
Why?

Reply:
In case I want to start the game with different parameters not yet implemented.
It just felt cleaner to me to have one part responsible for the launch arguments and another just for launching it.